packages/amplication-prisma-db/prisma/migrations/20240119173706_remove_btm_recommendations_tables/migration.sql

```diff
@@ -0,0 +1,21 @@
+/*
+  Warnings:
+
+  - You are about to drop the `BtmEntityRecommendation` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `BtmResourceRecommendation` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "BtmEntityRecommendation" DROP CONSTRAINT "BtmEntityRecommendation_btmResourceRecommendationId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "BtmEntityRecommendation" DROP CONSTRAINT "BtmEntityRecommendation_originalEntityId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "BtmResourceRecommendation" DROP CONSTRAINT "BtmResourceRecommendation_resourceId_fkey";
+
+-- DropTable
+DROP TABLE "BtmEntityRecommendation";
+
```

Comment:
Let's get rid of this migration and also delete the previous one where we initially created the table(assuming it was never deployed)

Reply:
It was deployed to sandbox

Reply:
Since this was deployed to sandbox, the overhead to change it manually is not worth it IMO. Let's stay with the migrations as-is.